tools/NuGet/template-nuget/DynamoVisualProgramming.Tests.nuspec

```diff
@@ -21,8 +21,8 @@
   </metadata>
   <files>
     <!--These assemblies are referenced from the bin directory since they are not harvested for the installer-->
-    <file src="..\..\..\bin\AnyCPU\Release\DynamoCoreTests.dll" target="lib\net47" />
-    <file src="..\..\..\bin\AnyCPU\Release\SystemTestServices.dll" target="lib\net47" />
-    <file src="..\..\..\bin\AnyCPU\Release\TestServices.dll" target="lib\net47" />
+    <file src="DynamoCoreTests.dll" target="lib\net47" />
+    <file src="SystemTestServices.dll" target="lib\net47" />
+    <file src="TestServices.dll" target="lib\net47" />
```

Comment:
How do these paths get resolved with this change?

Reply:
this is local to the build folder - most other nuspecs we have are like this - and I verified this works. I wanted to build a debug nuget locally and this stopped me.

Reply:
hmm - I verified the other package. Let me double check this one.

Reply:
@aparajit-pratap - it did work - but this may have only worked because I specified the harvest directory to pull files from as the bin folder...

not a big deal, I can revert this stuff for now.